### PR TITLE
Add XY Liquidity adapter

### PR DESCRIPTION
**Pull request by vickcharles**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### List of audit links if any: https://github.com/zokyo-sec/audit-reports/blob/main/Tren%20Finance/Tren%20Finance_Zokyo_audit_report_Nov19th_2024.pdf

Omniscia Tren Finance Audit

##### Logo (High resolution, will be shown with rounded borders):

pfp

##### Current TVL:
Not Applicable: We have not deployed mainnet

##### Treasury Addresses (if the protocol has treasury) 

0x727763929490cccf903c916589F3d33A95C5A266

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at : 
        
 
  > Please revert changes to package.json / package-lock.json

File: projects/tren-finance/index.js
Comment (g1nt0ki) [inline projects/tren-finance/index.js:1]: the tvl in uni v3 pool belongs to uniswap

**Comment by llamatester** [comment]
Error while running adapter at : 
        
 
  > Please revert changes to package.json / package-lock.json

**Comment by llamabutler** [comment]
The adapter at projects/tren-finance exports TVL: 
        
 
 
    arbitrum                  42
    
    total                    42.259098881999996

File: projects/tren-finance/index.js
Comment (vickcharles) [reply projects/tren-finance/index.js:1]: Correct, we intend to show our stable coin liquidity, not tvl @g1nt0ki

Comment (g1nt0ki) [reply projects/tren-finance/index.js:1]: sorry, we dont track that

**Pull request closed without merging**